Makefile

```diff
@@ -201,7 +201,7 @@ lint:
 # minikube
 LIVENESS_PROBE ?= true
 .PHONY: minikube-deploy
-minikube-deploy: minikube-start gadget-default-container kubectl-gadget
+minikube-deploy: gadget-default-container kubectl-gadget
```

Comment:
Why to remove this dependency? Pinging @mqasimsarfraz  and @blanquicet as I think you use it.